### PR TITLE
KAA-1173: Added information about system requirements

### PR DESCRIPTION
**Pull request by v-chyzhevskyi**
Added information about system requirements

### CODE REVIEW

File: doc/Administration-guide/System-installation/index.md [hunk 1]
```diff
@@ -5,39 +5,52 @@ permalink: /:path/
...
 This guide overview over kaa platform installation.
 
 ## System requirements
 
-To use Kaa, your system must meet the following minimum system requirements.
+To use Kaa, your system must meet the following minimum system requirements:
 
    * 64-bit OS
+   
+   for single kaa-node
```
Comment (ashvayka) [inline doc/Administration-guide/System-installation/index.md]: for single kaa-node service with all third-party services (SQL, NoSQL, Zookeeper, etc.) deployed remotely

File: doc/Administration-guide/System-installation/index.md [hunk 2]
```diff
@@ -5,39 +5,52 @@ permalink: /:path/
...
+   
+   * 256 Mb RAM
+   
+   for kaa-node with all third party components
```
Comment (ashvayka) [inline doc/Administration-guide/System-installation/index.md]: for kaa-node with all third party services deployed on the same node

**Comment by ashvayka** [comment]
Few minor comments, otherwise, LGTM.

**Comment by ashvayka** [comment]
Please add "Further reading" section

File: doc/Administration-guide/System-installation/index.md [hunk 3]
```diff
@@ -5,39 +5,52 @@ permalink: /:path/
...
-  * [AWS deployment](#aws-deployment)
-  * [Planning your deployment](#planning-your-deployment)
-* [Troubleshooting](#troubleshooting)
+
+* TOC
+{:toc}
 
 ## Introduction
 
 This guide overview over kaa platform installation.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md]: Is it better to say: "This guide provides an overview of Kaa platform installation."?

File: doc/Administration-guide/System-installation/index.md [hunk 4]
```diff
@@ -47,21 +60,31 @@ Kaa requires the following third party components to be installed and configured
...
 
 ### SQL database
 
-SQL database instance is used to store metadata about tenants, applications, endpoint groups, etc. This information is shared between endpoints, thus it's volume does not scale and it can be efficiently stored in modern SQL databases. To support high availability of Kaa cluster, SQL database should be also deployed in cluster mode.
+SQL database instance is used to store metadata about tenants, applications, endpoint groups, etc. 
+This information is shared between endpoints, thus it's volume does not scale and it can be efficiently stored in modern SQL databases. 
+To support high availability of Kaa cluster, SQL database should be also deployed in cluster mode.
 
-Kaa supports two SQL databases at the moment: PostgresSQL and MariaDB. If you planning to use kaa in a single node instance we recommend you to use PostgreSQL and MariaDB for multi node cluster because of better clusterization capabilities of MariaDB.
+Kaa supports two SQL databases at the moment: PostgresSQL and MariaDB. If you planning to use kaa in a single node instance we recommend you to use
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md]: If you plan to use Kaa

File: doc/Administration-guide/System-installation/index.md [hunk 5]
```diff
@@ -47,21 +60,31 @@ Kaa requires the following third party components to be installed and configured
 * [PostgreSQL 9.4](http://www.postgresql.org/download/) or [MariaDB 5.5](https://mariadb.org/download/) Kaa has been tested on the latest production release of PostgreSQL and MariaDB.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md]: Add a dot before: "Kaa has been tested on ..."

File: doc/Administration-guide/System-installation/index.md [hunk 6]
```diff
@@ -47,21 +60,31 @@ Kaa requires the following third party components to be installed and configured
...
 ### NoSQL database
 
-NoSQL database instance is used to store information about endpoint profiles, notifications, configurations, etc. The volume of this information scales linearly with amount of endpoints that are managed using particular Kaa cluster instance. NoSQL database nodes can be co-located with Kaa nodes on the same physical or virtual machines. Kaa support Apache Cassandra and MongoDB as a NoSQL database at the moment. The choose between MongoDB and Apache Cassandra depends only on your specific data analysis needs.
+NoSQL database instance is used to store information about endpoint profiles, notifications, configurations, etc. The volume of this information scales 
+linearly with amount of endpoints that are managed using particular Kaa cluster instance. 
+NoSQL database nodes can be co-located with Kaa nodes on the same physical or virtual machines. 
+Kaa support Apache Cassandra and MongoDB as a NoSQL database at the moment. 
+The choose between MongoDB and Apache Cassandra depends only on your specific data analysis needs.
 
 ## Installing Kaa
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md:90]: your Kaa server (91 line)

File: doc/Administration-guide/System-installation/index.md [hunk 7]
```diff
@@ -47,21 +60,31 @@ Kaa requires the following third party components to be installed and configured
...
+PostgreSQL and MariaDB for multi node cluster because of better clusterization capabilities of MariaDB.
 
 ### NoSQL database
 
-NoSQL database instance is used to store information about endpoint profiles, notifications, configurations, etc. The volume of this information scales linearly with amount of endpoints that are managed using particular Kaa cluster instance. NoSQL database nodes can be co-located with Kaa nodes on the same physical or virtual machines. Kaa support Apache Cassandra and MongoDB as a NoSQL database at the moment. The choose between MongoDB and Apache Cassandra depends only on your specific data analysis needs.
+NoSQL database instance is used to store information about endpoint profiles, notifications, configurations, etc. The volume of this information scales 
+linearly with amount of endpoints that are managed using particular Kaa cluster instance. 
+NoSQL database nodes can be co-located with Kaa nodes on the same physical or virtual machines. 
+Kaa support Apache Cassandra and MongoDB as a NoSQL database at the moment.
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md]: Kaa supports

File: doc/Administration-guide/System-installation/index.md [hunk 8]
```diff
@@ -47,21 +60,31 @@ Kaa requires the following third party components to be installed and configured
...
 * [Zookeeper 3.4.5](http://zookeeper.apache.org/doc/r3.4.5/). Kaa requires ZooKeeper for coordination of server components.
 
-Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 2.2.5](http://cassandra.apache.org/download/) as a NoSQL database. The installation steps for third-party components are provided in the following section.
+Kaa also requires [MongoDB 2.6.9](http://www.mongodb.org/downloads) or [Cassandra 3.5](http://cassandra.apache.org/download/) as a NoSQL database. 
+The installation steps for third-party components are provided in the following section.
 
 ### Zookeeper
 
-Apache ZooKeeper enables highly reliable distributed coordination of Kaa cluster nodes. Each Kaa node push information about connection parameters, enabled services and corresponding services load. Other Kaa nodes use this information in order to get list of their neighbors and communicate with them. Control service uses information about existing Bootstrap services and their connection parameters during SDK generation.
+Apache ZooKeeper enables highly reliable distributed coordination of Kaa cluster nodes. Each Kaa node push information about connection parameters,
```
Comment (Polinaa) [inline doc/Administration-guide/System-installation/index.md]: Each Kaa node pushes

**Pull request merged**